app/views/partials/_search_bar.html.haml

```diff
@@ -2,11 +2,13 @@
   .filters-bar
     .navbar
       .navbar-collapse.collapse
-        %ul.nav.navbar-nav
-          %li
+        .navbar-form
```

Comment:
This only centers the text? Why don't you use the `text-center` class from bootstrap?